package/MDAnalysis/coordinates/core.py

```diff
@@ -107,80 +124,84 @@ def get_writer_for(filename=None, format='DCD', multiframe=None):
     :class:`cStringIO.StringIO` instance then the *format* argument must be
     used.
 
-    :Arguments:
-      *filename*
-         The filename for the trajectory is examined for its extension and
-         the Writer is chosen accordingly.
-      *format*
-         If no *filename* is supplied then the format can be explicitly set;
-         possible values are "DCD", "XTC", "TRR"; "PDB", "CRD", "GRO".
-      *multiframe*
-         ``True``: write multiple frames to the trajectory; ``False``: only
-         write a single coordinate frame; ``None``: first try trajectory (multi
-         frame writers), then the single frame ones. Default is ``None``.
+    Parameters
+    ----------
+    filename : str
+        The filename for the trajectory is examined for its extension and
+        the Writer is chosen accordingly.
+    format : str
+        If no *filename* is supplied then the format can be explicitly set
+    multiframe : bool
+        ``True``: write multiple frames to the trajectory; ``False``: only
+        write a single coordinate frame; ``None``: first try trajectory (multi
+        frame writers), then the single frame ones. Default is ``None``.
 
     .. versionchanged:: 0.7.6
        Added *multiframe* keyword; the default ``None`` reflects the previous
        behaviour.
     """
-    from . import _trajectory_writers, _frame_writers
-
     if isinstance(filename, basestring) and filename:
```

Comment:
Depends I see these things as gradual maintainability fixes. So I'm in favor of introducing them along while we fix up the c-modules for python3. But I'm a little bit biased because I'm somewhat sure that I silently introduced this practice